templates/keycloak.service.erb

```diff
@@ -4,10 +4,16 @@ After=network.target
 
 [Service]
 Type=idle
-#Environment="JAVA_OPTS=-Xms1024m -Xmx20480m -XX:MaxPermSize=768m"
 <% if scope['keycloak::service_environment_file'] -%>
 EnvironmentFile=<%= scope['keycloak::service_environment_file'] %>
 <% end -%>
+<% if scope['keycloak::service_java_opts'] -%>
+  <% if scope['keycloak::server_java_opts'].is_a?(Array) -%>
```

Comment:
While the indentation is more readable, it will add unnecessary white space to the unit file.  I'd recommend removing white space before the condition portions of inner `if` statement.